ami-recycle-bin/main.tf

```diff
@@ -0,0 +1,144 @@
+provider "aws" {
+ region = "us-west-2"
+}
```

Comment:
Please explain to the user how a deployment to another region would work

Reply:
> Please explain to the user how a deployment to another region would work

- Updated the region as a variable
- Added comments for user to understand the significance of the provider block to provision resources in different regions